api/krusty/component_test.go

```diff
@@ -657,3 +657,63 @@ components:
 		})
 	}
 }
+
+func TestOrderOfAccumulatedComponent(t *testing.T) {
+	tests := map[string]struct {
+		input          []FileGen
+		expectedOutput string
+	}{
+		"a": {
```

Comment:
Can we rename the test to something more descriptive? For example, "after generators".

Reply:
Oh!
Sorry, I forgot to fix the test name.

Reply:
I fixed at https://github.com/kubernetes-sigs/kustomize/pull/5170/commits/0c38baae5e83ce8d377dac064f82d5130b4a80f1.